sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy

```diff
@@ -121,12 +121,14 @@ class APISpec extends Specification {
     /* Unignore any managed disk tests if a managed disk account is available to be tested. They are difficult to
      acquire so we do not run them in the nightly live run tests. */
     static def MANAGED_DISK_STORAGE = "MANAGED_DISK_STORAGE_"
+    static def VERSIONED_STORAGE = "VERSIONED_STORAGE_"
```

Comment:
This is for another Storage Account resource correct? This will need an update to the ARM template which deploys resources before live test runs for these new tests to work in the nightly live test run.

Reply:
@alzimmermsft that's correct. Versioning is feature that can be enabled/disabled. Few questions here:
- is there any documentation about how new resource should be added ? I guess it's not only ARM template.
- I'm going to check if versioning can be enabled via ARM at this point. If not then is there any alternative to inject account and it's credentials somehow into the pipeline?

Reply:
If there isn't a way to add this as part of ARM we could resort to using a static testing resource given the limited number of tests that'll be using it. Given this is a new service feature I have a feeling there won't be ARM support for this.

Reply:
Good news is that there is ARM support https://review.docs.microsoft.com/en-us/azure/storage/blobs/versioning-enable?branch=pr-en-us-104390&tabs=template .

Is the `azure-sdk-for-java\sdk\storage\test-resources.json` the only place that needs an update?

Reply:
On the other hand it might take some time before support through ARM is rolled out. 
@alzimmermsft what does it take to define static testing resource? Who does that?

Reply:
@JimSuplizio could you chime in on how to define static resource (hardcoded storage account in this case ) ?

Reply:
@kasobol-msft unfortunately, I'm not the right person for this. @alzimmermsft and @hemanttanwar are the ones that setup the templates and test resources and @danieljurek may also have some input here.